parsons/catalist/catalist.py

```diff
@@ -0,0 +1,388 @@
+"""Utilities for working with the Catalist Match API
+
+Install dependencies with `pip install parsons[catalist]`
+"""
+
+import base64
+import logging
+import os
+import tempfile
+import time
+import urllib
+from typing import Optional, Union, Dict, List
+from zipfile import ZipFile
+
+import requests
+from parsons.etl import Table
+from parsons.sftp import SFTP
+from parsons.utilities.api_connector import APIConnector
+
+logger = logging.getLogger(__name__)
+
+# Currently the default and only available template for working with the Match API
+template_id = 48827
+
+
+class CatalistMatch:
+    """Connector for working with the Catalist Match API.
+
+    The Catalist Match tool requires OAuth2.0 client credentials for the API as well as
```

Comment:
Maybe in the docs we can offer a broad overview of how to obtain client credentials from Catalist / what other setup steps are required

Reply:
Asking Catalist for this

Reply:
Slightly implemented with 74cd6c2924a9363b094065370780ece01c980b6c and I'm asking if they can provide any more info here